src/App.tsx

```diff
@@ -1,223 +1,116 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer/Footer';
+import { FilterValues } from './types/FilterValues';
+import { Errors } from './components/Errors';
+import { ErrorMessages } from './types/Error';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterValue, setFilterValue] = useState(FilterValues.All);
+  const [countActive, setCountActive] = useState(0);
```

Comment:
do you really need to store it in state?
you may calculate it and store in useMemo